src/payment/external-id-request-sender.ts

```diff
@@ -0,0 +1,25 @@
+import { RequestSender } from '@bigcommerce/request-sender';
+
+import { ContentType, INTERNAL_USE_ONLY } from '../common/http-request';
+
+export default class ExternalIdRequestSender {
+    constructor(
+        private _requestSender: RequestSender
+    ) {}
+
+    async saveExternalId(provider: string, externalId: string): Promise<void> {
```

Comment:
should this be named `methodId: string` rather than `provider` ? It's good to keep consistency of the naming of the variables, so it's easier to map out what's expected to be passed to this method.